modules/deployment-plug-in-cluster.adoc

```diff
@@ -25,6 +33,11 @@ Where:
 `--create-namespace`:: Optional: If deploying to a new namespace, use this parameter.
 `--set plugin.image=my-plugin-image-location`:: Specifies the location of the image within the `plugin.image` parameter.
 --
++
+[NOTE]
+====
+If you are deploying on {product-title} 4.10, it is recommended to exclude configurations related to pod security by adding the parameter `--set plugin.securityContext.enabled=false`.
```

Comment:
is this required `only` on 4.10 or 4.10 and later OCP?

Reply:
@spadgett I am not sure I know the answer to this. I thought it was only 4.10, but I am not sure.